helm/data_repository.go

```diff
@@ -40,6 +40,11 @@ func dataRepository() *schema.Resource {
 				Optional:    true,
 				Description: "Verify certificates of HTTPS-enabled servers using this CA bundle",
 			},
+			"insecure_skip_tls_verify": {
```

Comment:
Adding, new configuration a deprecated resource maybe it's not a good idea.

Reply:
I agree though I have a question. What are the plans for supporting helm repo advanced features in the future if not through a data resource? I'm thinking mostly about authentication which is most likely something folks use.

Reply:
The `repository` is pojo that contains that used to retrieve the package, all their options were available in the release call. But this one looks like it's [not available](https://github.com/helm/helm/blob/master/pkg/action/install.go#L114).

In this case, since we are removing `repository` no one needs to download the index with an invalid certificate. :rofl:

Reply:
Clear now, missed the fact you can already configure these though the release resource. The only inconvenience I see is when dealing with multiple releases from the same repo, there's going to be a lot of duplication, but that's going to be a separate conversation.

Should I revert the change? I actually didn't realize the repo data resource was marked deprecated before starting to work on my PR :)

Reply:
Yes, please remove this. We are not doing more updates on this resource. You can use vars to avoid duplication. `repository` has several problems but the main one is that is persisted on the client-side, causing a lot of pain to everyone.